compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -27,6 +27,7 @@ use rustc_middle::ty::codec::TyEncoder;
 use rustc_middle::ty::fast_reject::{self, TreatParams};
 use rustc_middle::{bug, span_bug};
 use rustc_serialize::{Decodable, Decoder, Encodable, Encoder, opaque};
+use rustc_session::config::enforceable_mitigations::DeniedPartialMitigation;
```

Comment:
Remove or rename enforceable_mitigations?

Reply:
renamed to mitigation_coverage